suave/devenv/docker-compose.yml

```diff
@@ -7,13 +7,18 @@ services:
       - --dev
       - --dev.gaslimit=30000000
       - --http
+      - --http.port=8545
+      - --http.addr=0.0.0.0
+      - --http.vhosts=*
+      - --http.corsdomain=*
       - --ws
+      - --ws.origins=*
+      - --ws.addr=0.0.0.0
       - --datadir=/data
       - --keystore=/keystore/keystore
       - --allow-insecure-unlock
       - --unlock=0xB5fEAfbDD752ad52Afb7e1bD2E40432A485bBB7F
       - --password=/keystore/password.txt
-      - --suave.eth.remote_endpoint=http://localhost:8555
```

Comment:
I got stuck in docker configurations here to connect both containers. If this flag is set, the `mev-share` example does not work because the `mevm` node cannot connect with the `suave-enabled` node. Planned as a followup task.